src/openzaak/components/documenten/openapi.yaml

```diff
@@ -2056,7 +2056,7 @@ components:
           maxLength: 200
           minLength: 1
         locked:
-          title: locked
+          title: geblokkeerd
```

Comment:
Was this change intentional? It seems that the Dutch description still says "gelocked"

Reply:
yeah not sure what's happening here, but re-generating the schema doesn't change the value. Looking at the po file, this does seem intentional:

```po
#: src/openzaak/components/documenten/admin.py:279
#: src/openzaak/components/documenten/api/serializers.py:226
msgid "locked"
msgstr "geblokkeerd"
```